photon-client/src/components/cameras/CamerasView.vue

```diff
@@ -146,6 +151,18 @@ th {
   justify-content: center;
 }
 
+@media only screen and (min-width: 960px) {
+  .fixed-right {
```

Comment:
Fixed right needs to be modified to cause the left column to expand itself such that the camera view is also right aligned cause currently, on large screens, there is a large gap that exists there, this also exists regardless of this CSS prop but...